src/UI/Components/ServiceInstanceForm/ServiceInstanceForm.tsx

```diff
@@ -150,18 +176,43 @@ export const ServiceInstanceForm: React.FC<Props> = ({
 
   return (
     <StyledForm onSubmit={preventDefault}>
-      {fields.map((field) => (
-        <FieldInput
-          key={field.name}
-          field={field}
-          formState={formState}
-          originalState={originalState}
-          getUpdate={getUpdate}
-          path={null}
-          suggestions={field.suggestion}
+      <ToggleGroup aria-label="form-editor-toggle-group">
+        <ToggleGroupItem
+          text="Form"
+          key={0}
+          buttonId="formButton"
+          isSelected={isForm}
+          isDisabled={!isEditorValid}
+          onChange={() => setIsForm(true)}
+        />
+        <ToggleGroupItem
+          text="JSON-Editor"
```

Comment:
is that displayed? if so we can put it to the words.ts

Reply:
done